flowers.schema.json

```diff
@@ -19,16 +19,19 @@
           "sci_name": {
             "type": "string"
           },
-          "image_file": {
-            "type": "string"
+          "art_file": {
```

Comment:
Will there always be just one art file? I think that seems sensible, but wanted to verify.